NDB.Covid19/NDB.Covid19.iOS/Views/InfectionStatus/InfectionStatusViewController.cs

```diff
@@ -197,7 +197,7 @@ async void SetStatusContainerState(bool isRunning)
             {
                 modalClosed = ModalViewController.IsBeingDismissed;
             }
-            if (NavigationController.TopViewController is InfectionStatusViewController && modalClosed)
+            if (NavigationController?.TopViewController is InfectionStatusViewController && modalClosed)
```

Comment:
This fixes a bug I found that caused a crash if you changed the language, immediately put the app in the background, and then open the app again.

Reply:
good finding!